document/src/test/java/com/yahoo/document/serialization/VespaDocumentSerializerTestCase.java

```diff
@@ -68,7 +68,6 @@ public void predicate_field_values_are_serialized() {
 
         CompressionFixture() {
             docType = new DocumentType("map_of_structs");
-            docType.contentStruct().setCompressionConfig(new CompressionConfig(CompressionType.LZ4));
```

Comment:
Should the test(s) that use this fixture be removed (or rewritten) since compression is now a no-op? (`compressed_map_of_compressed_structs_is_supported` at least)

Reply:
Correct, thanks. Done.